src/locales/en-US/list.ftl

```diff
@@ -2,10 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-## common
+## Firefox / Lockbox Brand
+##
+## Firefox and Lockbox must be treated as a brand, and kept in English.
+## They cannot be:
+## - Declined to adapt to grammatical case.
+## - Transliterated.
+## - Translated.
+##
+## Reference: https://www.mozilla.org/styleguide/communications/translation/
+-brand-name = Firefox
+-product-short-name = Lockbox
+-product-full-name = Firefox Lockbox
+# TODO: should Firefox Account also not be translated or declined?
+-product-accounts = Firefox Account
+
+header-logins-button = Logins
+
+profile-menu-account = Account
+profile-menu-sign-in = Sign in to Sync
```

Comment:
We should have a Sync term
https://hg.mozilla.org/l10n/gecko-strings/file/default/browser/browser/branding/sync-brand.ftl#l5

That would also avoid confusion when "sync" is just a verb and not the name of the feature